packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -1167,13 +1357,7 @@ export async function createHotReloaderTurbopack(
     clearHmrServerError() {
       // Not implemented yet.
     },
-    async start() {
-      const enabled = isTestMode || opts.telemetry.isEnabled
-      const nextVersionInfo = await getVersionInfo(enabled)
```

Comment:
This was already called earlier before returning the hotReloader interface, we don't have to run it twice.